app/helpers/application_helper.rb

```diff
@@ -16,4 +16,12 @@ def tooltip(text)
       'tabindex': '0'
     )
   end
+
+  def decorated_session
+    if @sp_name.present?
```

Comment:
Maybe for bonus points we could memoize the result of this so we don't make 2 allocations inside the `.slim` file? But really not a huge deal